src/gatling/resources/RecordedSimulationFRApplicant_0048_request.txt

```diff
@@ -0,0 +1,10 @@
+------WebKitFormBoundarynZAT1BNCygsLwdnc

```

Comment:
is this file needed?

Reply:
Yes, the file is needed and doesn't work if you put it directly inside the request.